e2e/tests/nightly/genomic-extraction-to-vcf.spec.ts

```diff
@@ -0,0 +1,230 @@
+import { findOrCreateWorkspace, signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import {
+  AnalysisTool,
+  Language,
+  LinkText,
+  ConceptSetSelectValue,
+  DatasetValueSelect,
+  AgeSelectionRadioButton
+} from 'app/text-labels';
+import CohortActionsPage from 'app/page/cohort-actions-page';
+import { makeRandomName, makeWorkspaceName } from 'utils/str-utils';
+import GenomicsVariantExtractConfirmationModal from 'app/modal/genomic-extract-confirmation-modal';
+import ExportToNotebookModal from 'app/modal/export-to-notebook-modal';
+import RuntimePanel, { ComputeType } from 'app/component/runtime-panel';
+import { logger } from 'libs/logger';
+import GenomicExtractionsSidebar from 'app/component/genomic-extractions-sidebar';
+import { Page } from 'puppeteer';
+import { takeScreenshot } from 'utils/save-file-utils';
+import expect from 'expect';
+
+// 80 minutes. Test could take a long time.
```

Comment:
Per other auth discussion, I guess 60m should really be the limit. The token would need to be refreshed to run longer than this.

Reply:
Makes sense. Changed back 60 min and added comments about refresh token.